### PR TITLE
resize lightbox graph

### LINKED ISSUE #7: Make graphs bigger on click.
When a user clicks a small graph, a lightbox appears showing a larger graph.

A better user experience would be for the graphs to fill more of the user's screen by stretching and/or requesting a larger image from ganglia.

### PR DESCRIPTION
**Pull request by AriettaZ**
#### full screen (1092*400):
full screen

#### smaller viewport (the same width as viewport): 
half screen

fix #7

### CODE REVIEW

File: vendor/assets/components/lightbox2/javascripts/lightbox.js [hunk 1]
```diff
@@ -22,8 +22,8 @@
       this.alwaysShowNavOnTouchDevices = false;
       this.fadeDuration                = 20;
       this.fitImagesInViewport         = true;
-      // this.maxWidth                    = 800;
-      // this.maxHeight                   = 600;
+      this.maxWidth                    = window.innerWidth;
+      this.maxHeight                   = window.innerHeight;
```
Comment (ericfranz) [inline vendor/assets/components/lightbox2/javascripts/lightbox.js]: We should not be modifying the default values in the library itself.

Rather, wherever we instantiate the lightbox, we should provide the "window.innerWidth" and "window.innerHeight" options i.e. https://lokeshdhakar.com/projects/lightbox2/index.html#options

**Comment by ericfranz** [comment]
TODO: Is this responsive, are there any problems before merging? Also need to fix the merge conflict.

**Comment by AriettaZ** [comment]
ezgif com-video-to-gif

**Comment by johrstrom** [comment]
Thanks for the PR, this is just great now.

**Pull request merged**